src/main/java/com/minecolonies/coremod/event/LootTableEventHandler.java

```diff
@@ -0,0 +1,38 @@
+package com.minecolonies.coremod.event;
+
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.storage.loot.LootPool;
+import net.minecraft.world.storage.loot.TableLootEntry;
+import net.minecraftforge.event.LootTableLoadEvent;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.minecolonies.api.util.constant.Constants.MOD_ID;
+
+public class LootTableEventHandler {
+    public static List<String> mobs = Arrays.asList(
+            "minecraft:entities/skeleton",
```

Comment:
We do not wish for the tome to be dropped by any entities besides the raiders, it adds a level of rarity to them, as well as requiring the player to endure these raids